### PR TITLE
Encoder documentation

### PR DESCRIPTION
**Pull request by chenkevinh**
Added documentation on configuring encoders and adding encoders to new boards/shields. Issue #140

### CODE REVIEW

File: docs/docs/dev-guide-new-shield.md [hunk 1]
```diff
@@ -195,6 +199,81 @@ Further documentation on behaviors and bindings is forthcoming, but a summary of
 - `trans` is the "transparent" behavior, useful to be place in higher layers above `mo` bindings to be sure the key release is handled by the lower layer. No binding arguments are required.
 - `mt` is the "mod-tap" behavior, and takes two binding arguments, the modifier to use if held, and the keycode to send if tapped.
 
+## Adding Features
+
+### Encoders
+
+EC11 encoder support can be added to your board or shield by adding the appropriate lines to your board/shield's configuration (.conf), device tree (.dtsi), and overlay (.overlay) files.
```
Comment (petejohanson) [inline docs/docs/dev-guide-new-shield.md:206]: It seems like we should also have a note/section that links to the encoders feature doc from here, so users adding the shield can add the default sensor bindings as well.

Thoughts?

**Review by petejohanson — COMMENTED** [review]
Just one thought, otherwise this looks good!

File: docs/docs/dev-guide-new-shield.md (hunk 1 above)
Comment (chenkevinh) [reply docs/docs/dev-guide-new-shield.md:206]: Yup good call, I'll add it as another tab to update the keymap as well

**Review by petejohanson — APPROVED** [review]
Thanks!

**Pull request merged**